.env.example

```diff
@@ -29,13 +30,17 @@ PEGGO_ETH_PASSPHRASE=
 PEGGO_ETH_PK=
 PEGGO_ETH_USE_LEDGER=false
 PEGGO_ETH_GAS_PRICE_ADJUSTMENT=1.3
+PEGGO_ETH_MAX_GAS_PRICE=500 // (GWei)
```

Comment:
Let's be specific and have string with denom there `500gwei`, and handle it as wei if no denom specified.

The approach of assuming Gwei could lead to millions of $ wasted 
https://etherscan.io/tx/0xca8f8c315c8b6c48cee0675677b786d1babe726773829a588efa500b71cbdb65